lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -888,6 +888,56 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		return acc;
 	});
 
+	/**
+	* Returns a new typed array containing the elements in reversed order.
+	*
+	* @name toReversed
+	* @memberof TypedArray.prototype
+	* @type {Function}
+	* @throws {TypeError} `this` is not a typed array
+	* @returns {TypedArray} reversed array
+	*
+	* @example
+	* var Float64ArrayFE = factory( 'float64' );
+	*
+	* var arr = new Float64ArrayFE( 'little-endian', [1.0, 2.0, 3.0] );
+	*
+	* var out = arr.toReversed();
+	* // returns <TypedArray>
+	*
+	* var v = out.get( 0 );
+	* // returns 1.0
+	*
+	* v = out.get( 1 );
+	* // returns 2.0
+	*
+	* v = out.get( 2 );
+	* // returns 3.0
+	*/
+	setReadOnly( TypedArray.prototype, 'toReversed', function toReversed() {
+		var obuf;
+		var out;
+		var len;
+		var buf;
+		var i;
+		var v;
+
+		if ( !isTypedArray( this ) ) {
+			throw new TypeError( 'invalid invocation. `this` is not a typed array.' );
+		}
+
+		len = this._length;
+		out = new this.constructor( flag2byteOrder( this._isLE ), len );
+		buf = this._buffer;
+		obuf = out._buffer; // eslint-disable-line no-underscore-dangle
+		for ( i = 0; i < len; i++ ) {
+			v = buf[ GETTER ]( ( ( len - 1 ) * BYTES_PER_ELEMENT ) - ( i * BYTES_PER_ELEMENT ), this._isLE );
```

Comment:
You're doing more arithmetic than is necessary.

```suggestion
			v = buf[ GETTER ]( ( len - i - 1 ) * BYTES_PER_ELEMENT, this._isLE );
```